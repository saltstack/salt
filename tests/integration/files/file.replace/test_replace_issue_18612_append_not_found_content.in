#john=doe
#foo = bar
#foo=bar
#blah=blub
#some=thing
