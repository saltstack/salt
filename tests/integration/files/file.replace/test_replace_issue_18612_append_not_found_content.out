#john=doe
#foo = bar
#foo=bar
#blah=blub
#some=thing
THIS LINE WASN'T FOUND! SO WE'RE APPENDING IT HERE!
